Update online directories

SpendingBitcoin opened the pull request:
I have replaced outdated links with https://spending-bitcoin.com, an online directory with 745 active online stores accepting Bitcoin (and still growing). Spending-Bitcoin.com isn't monetized so no advertisements, no hidden referral links, no tracking/storing user data etc.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories + small fix

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin committed:
Update online directories

Replaced outdated link with Spending-Bitcoin.com, a directory with over 745 active online stores.

SpendingBitcoin commented:
Pull request is visible again. Our account got flagged: https://twitter.com/SpendingBitcoin/status/1194548884309184513

wbnns commented:
@SpendingBitcoin 

Thanks for the recommendation.

Since the website has only been up for a [short period of time](https://www.reddit.com/r/Bitcoin/comments/ds6lot/weve_launched_spendingbitcoincom_the_biggest/), we would recommend waiting at least a few months and then proceeding to review the site for potential inclusion, to see how the quality of the listings hold up.

If other contributors disagree or recommend otherwise in the interim, we can reopen.

Also, unrelated, every page on the site (750+) has the same title tag and meta description. This could lead to issues with the site getting removed or not appearing in search engine indexes.

SpendingBitcoin commented:
Small update:
- Drastically improved server response time
- Improved mobile support
- Added more online stores (currently at 928 online stores)
- Added by Wallet of Satoshi: https://twitter.com/walletofsatoshi/status/1199462392737886213

wbnns commented:
Thanks for letting us know.

SpendingBitcoin commented:
Hello! It's been a while since we created our pull request. There are currently over 1,100 online stores accepting BTC listed on our website and we're still increasing those numbers every day! As we initially envisioned, our website:

- Is BTC only (no shitcoins)
- Has no paid listings
- Has no advertisements
- Has no referral/affiliate links
- Does not store any user data
- Does not user trackers

We would be incredibly happy if you're willing to reconsider our pull request!

wbnns commented:
@SpendingBitcoin 

Thank you for the update. We've reopened the pull request and will take another look.

Could you please fix the merge conflicts on this PR in the mean time?

SpendingBitcoin commented:
That's great! We'll take care of the merge conflicts. Thanks!

wbnns commented:
Following up on this, we've took a random sample of a dozen listings and encountered a couple that do not accept bitcoin:
+ https://spending-bitcoin.com/entry/cord-shoes-and-boots-made-in-atlanta-georgia-usa
+ https://spending-bitcoin.com/entry/jupityr-premium-watches-sunglasses-accessoires

With respect to this specific PR, we're not sure it would be fair to 99bitcoins, to remove/replace them on the site, when their page (that is currently linked, and has been linked to for years) seems to have been recently updated as of approximately a month ago.

That said, on the topic of linking to sites that help people discover where they can spend bitcoin, we know that any directory with a large number of options is going to have regular maintenance challenges that ensure the list of options are accurate, 99bitcoins included.

We should consider if we should specifically have a hyperlink to one or more "online directories" on the [Spend Bitcoin](https://bitcoin.org/en/spend-bitcoin) page, due to this issue.

SpendingBitcoin commented:
Hey, sorry for the late reply. Thanks for letting us know about those submissions. The top one still mentions Bitcoin payments on their website but they appear to be updating their web shop. 

We do daily random sampling to see if everything is up-to-date but, as you said, some will slip through which is why users can report them too.

Regarding 99bitcoins, we're not trying to steal anyone's spotlight. Our only goal is to spread Bitcoin adoption and bring more traffic to merchants who are supporting the Bitcoin ecosystem. Adding multiple links/websites on the Spend Bitcoin page seems like a great option!

wbnns commented:
@SpendingBitcoin 

Understood, thanks for following up.

TheTmelt commented:
> I have replaced outdated links with https://spending-bitcoin.com, an online directory with 745 active online stores accepting Bitcoin (and still growing). Spending-Bitcoin.com isn't monetized so no advertisements, no hidden referral links, no tracking/storing user data etc.

Hi I know I am a random person, but i was wondering if you have implemented this concept of spendbitcoin to the atm's and of small cash avenues? Like maybe cash back at stores or "round up total" sending the little extra to a wallet.. I just wanted to help  and know if i did at all. Also if you can message me on the or this idea. :)
Ty
